editor/src/components/canvas/commands/show-grid-controls-command.ts

```diff
@@ -1,27 +1,30 @@
 import type { ElementPath } from 'utopia-shared/src/types'
 import type { BaseCommand, CommandFunction, WhenToRun } from './commands'
-import type { EditorState, EditorStatePatch } from '../../editor/store/editor-state'
+import type { EditorState, EditorStatePatch, ValidOrInvalid } from '../../editor/store/editor-state'
 import type { GridCellCoordinates } from '../canvas-strategies/strategies/grid-cell-bounds'
 
 export interface ShowGridControlsCommand extends BaseCommand {
   type: 'SHOW_GRID_CONTROLS'
   target: ElementPath
   targetCell: GridCellCoordinates | null
   rootCell: GridCellCoordinates | null
+  rootCellTargetValidity: ValidOrInvalid
```

Comment:
Do we really need this? We just use this to not highlight the target cell, but we could just set the target cell to zero in those cases